app.py

```diff
@@ -1,59 +1,83 @@
-        query_url = TRELLO_URL_BASE + 'cards'
-        params = {'name': card_title, 'idList': get_todo_list_id()}
-        params.update(DEFAULT_PARAMS)
-        requests.post(query_url, data=params).json()
+        card = {
+            'name': card_title,
+            'dateLastActivity': datetime.datetime.now().isoformat()
+        }
+        devops_database = mongo_manager.get_database()
+        inserted_id = devops_database[board_utils.MONGO_LIST_TODO].insert_one(card).inserted_id
+        print('Created card with ID: ' + str(inserted_id))
         return redirect("/")
 
     @app.route('/completeditem', methods=['POST'])
     def updateListItem():
-        print("Updating Item!")
+        print('Updating Item!')
         print(request.form.get('id'))
         card_id = request.form.get('id')
-        query_url = TRELLO_URL_BASE + 'cards/' + card_id
-        params = {'idList': get_done_list_id()}
-        params.update(DEFAULT_PARAMS)
-        requests.put(query_url, data=params)
-        return redirect("/")
+        card_old_list = board_utils.MONGO_LIST_TODO
 
-    def get_todo_list_id():
-        for trello_list in all_lists:
-            if trello_list['name'] == 'Things To Do':
-                return trello_list['id']
+        devops_database = mongo_manager.get_database()
+
+        completed_card = devops_database[board_utils.MONGO_LIST_TODO].find_one({'_id': ObjectId(card_id)})
+        if completed_card is None:
+            card_old_list = board_utils.MONGO_LIST_DOING
+            completed_card = devops_database[board_utils.MONGO_LIST_DOING].find_one({'_id': ObjectId(card_id)})
+            if completed_card is None:
+                print('No card to update with ID: ' + card_id)
+                return redirect("/")
+        devops_database[card_old_list].delete_one({'_id': completed_card['_id']})
+        devops_database[board_utils.MONGO_LIST_DONE].insert_one(completed_card)
+        return redirect("/")
 
-    def get_done_list_id():
-        for trello_list in all_lists:
-            if trello_list['name'] == 'Done':
-                return trello_list['id']
+    def get_all_cards():
+        devops_database = mongo_manager.get_database()
+        to_do_items = devops_database['to_do'].find()
+        doing_items = devops_database['doing'].find()
+        done_items = devops_database['done_items'].find()
+        return {
+            board_utils.MONGO_LIST_TODO: to_do_items,
+            board_utils.MONGO_LIST_DOING: doing_items,
+            board_utils.MONGO_LIST_DONE: done_items
+        }
 
     return app
 
 
+class MongoConnectionManager:
+    MONGO_PASS = board_utils.MONGO_PASS
+    MONGO_USER = board_utils.MONGO_USER
+
+    mongo_client = None
+
+    def get_database(self):
+        if self.mongo_client is None:
+            self.mongo_client = pymongo.MongoClient(
+                "mongodb+srv://" + self.MONGO_USER + ":" + self.MONGO_PASS + "@cluster0.wyf78.mongodb.net/DevopsEx?retryWrites=true&w=majority")
```

Comment:
Would be good to make the host or even the whole URL an environment variable so it's easier for other people to run

Reply:
True, might add in a subsequent PR